src/sparsezoo/model/utils.py

```diff
@@ -120,31 +139,29 @@ def load_files_from_stub(
     models = api.fetch(
         operation_body="models",
         arguments=arguments,
-        fields=["modelId", "modelOnnxSizeCompressedBytes"],
+        fields=[
+            "model_id",
+            "model_onnx_size_compressed_bytes",
+            "files",
+            "benchmark_results",
+            "training_results",
+        ],
     )
 
     if len(models):
```

Comment:
Am I grumpy old man for wanting to raise an error if there are many matching models instead of just rolling with the first model in the list? I don't know what the right answer is.